plume-models/src/reshares.rs

```diff
@@ -143,7 +143,7 @@ impl FromId<PlumeRocket> for Reshare {
                 post_id: Post::from_id(
                     c,
                     &{
-                        let res: String = act.announce_props.object_link::<Id>()?.into();
+                        let res = act.announce_props.object_link::<Id>()?;
```

Comment:
I think you can even replace this block with just `&act.announce_props.object_link::<Id>()?`, it was only here to be able to convert the Id to a String (and same for other similar blocks).